src/compiler/checker.ts

```diff
@@ -39494,6 +39494,8 @@ namespace ts {
 
             // NOTE: assignability is checked in checkClassDeclaration
             const baseProperties = getPropertiesOfType(baseType);
+            const derivedClassDecl = getClassLikeDeclarationOfSymbol(type.symbol)!;
+            const inheritedAbstractMemberNotImplementedErrors: Diagnostic[] = [];

```

Comment:
instead of saving a list, create an error lazily and push onto that. It already has a list internally.